EXPERIMENTAL: make check for GLEW_ARB_geometry_shader4 optional, to
allow testing on integrated Intel GPU

Index: rts/Rendering/GlobalRendering.cpp
--- rts/Rendering/GlobalRendering.cpp.orig
+++ rts/Rendering/GlobalRendering.cpp
@@ -677,7 +677,8 @@ void CGlobalRendering::CheckGLExtensions() const
 	CHECK_OPT_EXT(GLEW_ARB_shading_language_100); // 2.0
 	CHECK_REQ_EXT(GLEW_ARB_vertex_shader); // 1.5 (VS-GLSL; core in 2.0)
 	CHECK_REQ_EXT(GLEW_ARB_fragment_shader); // 1.5 (FS-GLSL; core in 2.0)
-	CHECK_REQ_EXT(GLEW_ARB_geometry_shader4); // GS v4 (GL3.2)
+	//CHECK_REQ_EXT(GLEW_ARB_geometry_shader4); // GS v4 (GL3.2)
+	CHECK_OPT_EXT(GLEW_ARB_geometry_shader4); // GS v4 (GL3.2)
 
 	#undef CHECK_OPT_EXT
 	#undef CHECK_REQ_EXT
